src/kernels/raw/session/rawJupyterSession.node.ts

```diff
@@ -106,17 +106,6 @@ export class RawJupyterSession extends BaseJupyterSession implements IRawKernelC
                 sendKernelTelemetryEvent(this.resource, Telemetry.RawKernelSessionStartUserCancel);
                 traceVerbose('Starting of raw session cancelled by user');
                 throw error;
-            } else if (error instanceof TimedOutError) {
```

Comment:
`TimeoutError` class was never used anywhere, meaning we'd never throw such an error anywhere.